keras-runner/src/main/kotlin/org/jetbrains/research/keras/runner/deserializer/json/LayerConfig.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.research.keras.runner.deserializer.json
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class LayerConfig {
+    abstract var name: String
+    abstract var trainable: Boolean
+    abstract var batchInputShape: List<Int?>?
```

Comment:
I would make it private and add val that is `batchInputShape.filterNotNul()` -- public